tests_v2/test_workspace_icon_actions.py

```
@@ -0,0 +1,114 @@
+            cli_set = _run_cli_json(
+                cli,
+                ["workspace-action", "--workspace", ws_id, "--action", "set-icon", "--icon", symbol_json],
+            )
+            expected_symbol = {"type": "symbol", "name": "folder.fill"}
+            _must(cli_set.get("custom_icon") == expected_symbol, f"CLI set-icon should return symbol payload: {cli_set}")
+            _must(_workspace_icon(c, ws_id) == expected_symbol, "CLI set-icon should persist through socket state")
+        finally:
+            try:
+                c.close_workspace(ws_id)
+            except Exception:
+                pass
```

Ruff (0.15.12)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(Exception): ...`

(SIM105)